src/components/pages/Header.js

```diff
@@ -0,0 +1,21 @@
+import './Style.css';
+
+const Header = () => (
+  <header>
+    <nav>
+      <h1>Math Magicians</h1>
+      <ul>
+        <li>
+          <a href="/">Home</a>
+        </li>
+        <li>
+          <a href="/calculator">Calculator</a>
+        </li>
+        <li>
+          <a href="/quote">Quote</a>
```

Comment:
- [ ] [Optional] It would be better professionally if you replace the regular anchor tag `<a>` with the `<Link>` tag from `react-router-dom` for more info about why you do this change kindly see that [link](https://reactrouter.com/en/main/start/tutorial#client-side-routing).